src/3D/3D_elliptic.F90

```diff
@@ -502,8 +503,13 @@ subroutine generateSolution
   call KSPSetTolerances(ellipKSP, kspRelTol, 1e-16, 1e20, kspMaxIts, ierr)
   call EChk(ierr, __FILE__, __LINE__)
 
+#if PETSC_VERSION_GT(3, 14, 0)
```

Comment:
I believe the correct version check would be `PETSC_VERSION_GE(3, 15, 0)` since this renaming happened in 3.15

Reply:
Ah, I just thought we were checking greater than 3.14, but this makes sense